.github/workflows/ci.yml

```
@@ -54,6 +67,18 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
```

```suggestion
        uses: actions/checkout@v3
        with:
          fetch-depth: 0
```

otherwise line 79-81 don't work